tests/checks/create-base-directories.fish

```diff
@@ -11,7 +11,15 @@ $fish -c ''
 # Check that existing directories kept their permissions, and new directories
 # have the right permissions according to the XDG Base Directory Specification.
 # Use command ls and awk to strip off xattr or SELinux indicators.
-command ls -ld $dir/old $dir/old/new $dir/old/new/fish | awk '{print substr($1, 1, 10)}'
+command ls -ld $dir/old | awk '{print substr($1, 1, 10)}'
 # CHECK: drwxr-xr-x
+
+set -l ls_result (command ls -ld $dir/old/new $dir/old/new/fish | awk '{print substr($1, 1, 10)}' | string split0)
```

Comment:
I'd use a quoted command substitution instead of `string split0` because the intent isn't to split on NUL - `"$(command ls ...)"`